lib/rules/indent_size.ts

```diff
@@ -5,14 +5,25 @@ import EditorConfigError =  require('../editor-config-error');
 
 var LEADING_SPACES_MATCHER = /^ +/;
 
+function getNumber(num: string | number, fallback?: Function) {
+	if (typeof num === 'number') {
+		return num;
+	}
+
+	num = parseInt(num, undefined);
```

Comment:
Why pass `undefined`? Shouldn't this be `10`? Also, did you consider a `JSON.parse(num)`?